MCprep_addon/mcprep_ui.py

```diff
@@ -1969,6 +1984,25 @@ class McprepProps(bpy.types.PropertyGroup):
 	effects_list_index: bpy.props.IntProperty(default=0)
 
 
+class MCprepWindowManager(spawn_util.VariationProp, bpy.types.PropertyGroup):
```

Comment:
I might also need some additional context, where and how is this manager window being used? Is this mean to be just open session property settings used to drive operator functions?

If so, might warrant some more discussion on how I'm thinking things will evolve, since we're moving towards things like resource pack layering

Reply:
Yes, since this is more of an UI properties that need only one, single to operate and doesn't need to save to the context scene, blend file so WindowManager would be the best place for it.